app/controllers/comments_controller.rb

```diff
@@ -12,6 +14,12 @@ def create
     redirect_to user_post_path(current_user.id, params[:post_id])
   end
 
+  def destroy
+    @comment = Comment.find(params[:id])
+    @comment.destroy
+    redirect_to user_post_path(current_user.id, params[:post_id])
+  end
```

Comment:
- [ ] It would be nice if you check the delete functionality because I am not able to delete my own created comments :fearful: Kindly fix this issue to make your app work as expected :+1: Follow the steps to see this problem :blush:

1. First, create a comment on a post then you will see a delete button :+1:
2. But sign out and then sign in
3. Now go to the same post where you have commented
4. Now you won't see any delete button
5. Now add a comment again on the same post
6. Now you will see all the delete buttons :fearful: